advanced_tutorials/air_quality/feature_pipeline.py

```diff
@@ -149,12 +149,10 @@ def parse_weather(last_dates_dict, today):
     df_aq_update.date = df_aq_update.date.astype(str)
     df_weather_update.date = df_weather_update.date.astype(str)
 
-    air_quality_fg.insert(df_aq_update,
-                          write_options={'wait_for_job': False})
+    air_quality_fg.insert(df_aq_update)
     print("Created job to insert parsed PM2.5 data into FS...")
     print("Inserting into air_quality fg.")
 
-    weather_fg.insert(df_weather_update,
-                      write_options={'wait_for_job': False})
+    weather_fg.insert(df_weather_update, wait=True)
```

Comment:
It doesn't matter for this one. 
We are not consistent. Some feature pipelines always have wait=True for the last insertion - so that the data will be there when you start the training pipeline. 

I don't think we need to wait. In my experience it's rare somebody is fast enough to start the training pipeline fast enough.